1.5.0

```diff
@@ -0,0 +1,63 @@
+{
+   "instructionList":[
+      {
+         "instructionTypeName":"AC Basic User",
+         "taskList":[
+            {
+               "jiraTicket":"AC-26710",
```

Comment:
Need to label these with a, b, c etc so we can relate the log to the task easier.